scripts/hail_batch/variant_selection/main.py

```diff
@@ -20,7 +20,7 @@
     batch,
     f'hgdp_1kg_tob_wgs_variant_selection.py --output={OUTPUT}',
     max_age='12h',
-    num_secondary_workers=20,
+    num_workers=50,
```

Comment:
Should we try 20 first?

Reply:
oops, just saw this. I can send a new PR for 20 instead